lib/dpl/provider/bintray.rb

```diff
@@ -0,0 +1,532 @@
+require 'json'
+require 'net/http'
+require "uri"
+require 'find'
+
+module DPL
+  class Provider
+    class Bintray < Provider
+
+      @file
+      @user
+      @key
+      @passphrase
+      @url
+      @descriptor
+      @dry_run
+      @test_mode = false
+
+      def check_auth
+      end
+
+      def needs_key?
+        false
+      end
+
+      def set_test_mode
+        @test_mode = true
+      end
+
+      def init_from_args
+        @user = options[:user]
+        @key = options[:key]
+        @url = options[:url]
+        @file = options[:file]
+        @passphrase = options[:passphrase]
+        @dry_run = options[:dry_run]
+
+        if @user.nil?
```

Comment:
Please let me know if I'm missing something.
I did the following test:
```
@user = options[:user]
str = "false"
bool = false

if str == @user
   puts '1'
end
if bool == @user
   puts '2'
end
```
The output I git is 1.
What do you think? Can we leave the code as is?